### PR TITLE
Added bbox, center and polyline decode - (2nd attempt, from a local branch).

### PR DESCRIPTION
**Pull request by armantorkzaban**
Carries on the great work of @Dennis-Mwea (#55).

### CODE REVIEW

File: test/components/center_test.dart [hunk 1]
```diff
@@ -0,0 +1,52 @@
+
```
Comment (lukas-h) [inline test/components/center_test.dart:1]: This is actually the only thing that keeps it from being merged. Please implement the test before you work on any other PR. Thank you @armantorkzaban

**Review by lukas-h — CHANGES_REQUESTED** [review]
Please implement unit tests for center function

File: test/components/center_test.dart (hunk 1 above)
Comment (armantorkzaban) [reply test/components/center_test.dart:1]: The test is already there, implemented. Needs Bbox though, in order to work.

Comment (armantorkzaban) [reply test/components/center_test.dart:1]: I think at this point we can merge the two branches and deal with them as one. @lukas-h, what do you think?

Comment (lukas-h) [reply test/components/center_test.dart:1]: We could also try to combine the two branches to one

Comment (armantorkzaban) [reply test/components/center_test.dart:3]: What do you mean by 'combining'? Do you mean here on GH or do you mean merging? or both?

**Comment by lukas-h** [comment]
closed in favor of #99

**Pull request closed without merging**